src/Db/Diff/PKSimDB_diff.txt

```diff
@@ -1131,6 +1131,8 @@ UPDATE tab_calculation_method_rates SET formula='TM * kcat * max(EffectiveLumenC
 UPDATE tab_calculation_method_rates SET formula='TM * kcat * EffectiveLumenConcentration / (Km+EffectiveLumenConcentration)' WHERE calculation_method='LinksCommon' AND formula_rate='MucosaActiveInfluxFromLumen_MM';
 INSERT INTO tab_calculation_methods(calculation_method,display_name,description,category,sequence) VALUES('ExpressionParameters','Expression parameters','Calculation of expression parameters for individual molecules','ExpressionParameters',1);
 INSERT INTO tab_categories(category,display_name,description,category_type,icon_name) VALUES('ExpressionParameters','Expression parameters','Expression parameters','Individual',NULL);
+INSERT INTO tab_container_names(container_type,container_name,display_name,description,extension,sequence,icon_name) VALUES('PROCESS','LiverActiveEffluxToDuodenum_FirstOrder','Biliary secretion to the lumen/duodenum - First Order','Active transport process from the hepatocyte into the lumen duodenum','_lnk',1,'Gallbladder');

```

Comment:
@Yuri05 I am going to replace the current process for biliary excretion with our standard process. I don't see why we need something specific here as long as we support first order